perfdash/config.go

```diff
@@ -340,6 +340,45 @@ var (
 					Parser:           parsePerfData,
 				}},
 		},
+		"Network_Baseline": {
```

Comment:
Please split perf-dash changes to a separate PR.

Reply:
yes created another PR.